Scenarios/TripPin/src/webapi/ODataSamples.WebApiService/Controllers/PeopleController.cs

```diff
@@ -27,6 +27,7 @@ public IHttpActionResult Get()
         }
 
         // GET odata/People('key')
+        [EnableQuery]
```

Comment:
No docs yet.  Normally, it's developer's responsibility in Get() method to apply the query options one by one if he uses the ODataQueryOptions<T> as argument, otherwise he should add [EnableQuery] attribute on the action and let Web API OData to apply the query options. 

However, from the implementation of Get() method, it only applies the IfNonMatch, so it's right to add the [EnableQuery] on the method to allow other query options. I will merge it. Thanks.


---
In reply to: [63642434](https://github.com/OData/ODataSamples/pull/39#discussion_r63642434) [](ancestors = 63642434)